src/Mod/PartDesign/Gui/TaskPocketParameters.cpp

```diff
@@ -426,25 +447,35 @@ void TaskPocketParameters::onDirectionCBChanged(int num)
     if (axesInList.empty() || !pcPocket)
         return;
 
+    // we use this scheme for 'num'
+    // 0: normal to sketch or face
+    // 1: selection mode
+    // 2: custom
+    // 3-x: edges selected in the 3D model
+
+    // check the axis
+    // when the link is empty we are either in selection mode
+    // or we are normal to a face
     App::PropertyLinkSub& lnk = *(axesInList[num]);
-    if (lnk.getValue() == 0) {
+    if (num == 1) {
         // enter reference selection mode
         this->blockConnection(false);
         // to distinguish that this is the direction selection
         selectionFace = false;
         TaskSketchBasedParameters::onSelectReference(true, true, false, true, true);
         return;
     }
-    else {
+    else if (lnk.getValue() != 0) {
         if (!pcPocket->getDocument()->isIn(lnk.getValue())) {
             Base::Console().Error("Object was deleted\n");
             return;
         }
         propReferenceAxis->Paste(lnk);
-        // in case user is in selection mode, but changed his mind before selecting anything
-        exitSelectionMode();
     }
 
+    // in case user is in selection mode, but changed his mind before selecting anything

```

Comment:
> but changed `their` mind

Reply:
Also on 472 of TaskPadParameters.cpp

Reply:
> > but changed `their` mind

'user is' is singular, however it is just a comment and its meaning is clear

Reply:
Clear, and sexist. If you're looking for an authority, the AP style guide permits the singular "their": "They/them/their is acceptable in limited cases as a singular and-or gender-neutral pronoun, when alternative wording is overly awkward or clumsy."

Reply:
> Clear, and sexist.

What? "The user changed his mind." is a sexism sentence?
I requested you to review the PR according to the code and not about linguistics.
I prefer writing more comments than others because I experience often how difficult it is when I look at code others wrote. Comments help a lot and the comment does what it should - provide the info why the code is there at this position.

Reply:
Maybe I overreacted bit. However, I only moved an existing comment. I am also no native speaker and have no clue what 'AP' stands for.

Reply:
I've noticed this tendency in non-native speakers translating their thoughts in to english, that unknowingly assign gender. It's also baked-in to the native language as well, I catch myself ascribing 'default' masculine attributes when none are necessary. That's why there is an effort to use more gender neutral language in documentation and the UI/UX.

TIL that AP stands for Associated Press stylebook "...provides an A-Z guide to issues such as capitalization, abbreviation, punctuation, spelling, numerals and many other questions of language usage."